java-util/src/test/java/io/druid/java/util/common/parsers/TimestampParserTest.java

```diff
@@ -38,125 +40,97 @@ public void testStripQuotes() throws Exception
     Assert.assertEquals(expectedDt, parser.apply("0"));
     Assert.assertEquals(expectedDt, parser.apply("0000"));
     Assert.assertEquals(expectedDt, parser.apply(999999L));
   }
 
-  /*Commenting out until Joda 2.1 supported
   @Test
-  public void testTimeStampParserWithQuotes() throws Exception {
+  public void testTimeStampParserWithQuotes() throws Exception
+  {
     DateTime d = new DateTime(1994, 11, 9, 4, 0, DateTimeZone.forOffsetHours(-8));
-    Function<String, DateTime> parser = ParserUtils.createTimestampParser("EEE MMM dd HH:mm:ss z yyyy");
+    Function<String, DateTime> parser = TimestampParser.createTimestampParser("EEE MMM dd HH:mm:ss z yyyy");
     Assert.assertEquals(d.getMillis(), parser.apply(" \" Wed Nov 9 04:00:00 PST 1994 \"  ").getMillis());
   }
 
   @Test
-  public void testTimeStampParserWithShortTimeZone() throws Exception {
+  public void testTimeStampParserWithShortTimeZone() throws Exception
+  {
     DateTime d = new DateTime(1994, 11, 9, 4, 0, DateTimeZone.forOffsetHours(-8));
-    Function<String, DateTime> parser = ParserUtils.createTimestampParser("EEE MMM dd HH:mm:ss z yyyy");
+    Function<String, DateTime> parser = TimestampParser.createTimestampParser("EEE MMM dd HH:mm:ss z yyyy");
     Assert.assertEquals(d.getMillis(), parser.apply("Wed Nov 9 04:00:00 PST 1994").getMillis());
   }
 
   @Test
-  public void testTimeStampParserWithLongTimeZone() throws Exception {
+  public void testTimeStampParserWithLongTimeZone() throws Exception
+  {
 
     long millis1 = new DateTime(1994, 11, 9, 4, 0, DateTimeZone.forOffsetHours(-8)).getMillis();
     long millis2 = new DateTime(1994, 11, 9, 4, 0, DateTimeZone.forOffsetHours(-6)).getMillis();
 
-    Function<String, DateTime> parser = ParserUtils.createTimestampParser("EEE MMM dd HH:mm:ss zZ z yyyy");
+    Function<String, DateTime> parser = TimestampParser.createTimestampParser("EEE MMM dd HH:mm:ss zZ z yyyy");
     Assert.assertEquals(millis1, parser.apply("Wed Nov 9 04:00:00 GMT-0800 PST 1994").getMillis());
     Assert.assertEquals(millis2, parser.apply("Wed Nov 9 04:00:00 GMT-0600 CST 1994").getMillis());
     Assert.assertEquals(millis1, parser.apply("Wed Nov 9 04:00:00 UTC-0800 PST 1994").getMillis());
     Assert.assertEquals(millis2, parser.apply("Wed Nov 9 04:00:00 UTC-0600 CST 1994").getMillis());
 
-    parser = ParserUtils.createTimestampParser("EEE MMM dd HH:mm:ss zZ yyyy");
+    parser = TimestampParser.createTimestampParser("EEE MMM dd HH:mm:ss zZ yyyy");
     Assert.assertEquals(millis1, parser.apply("Wed Nov 9 04:00:00 GMT-0800 1994").getMillis());
     Assert.assertEquals(millis2, parser.apply("Wed Nov 9 04:00:00 GMT-0600 1994").getMillis());
     Assert.assertEquals(millis1, parser.apply("Wed Nov 9 04:00:00 UTC-0800 1994").getMillis());
     Assert.assertEquals(millis2, parser.apply("Wed Nov 9 04:00:00 UTC-0600 1994").getMillis());
-
-    parser = ParserUtils.createTimestampParser("EEE MMM dd HH:mm:ss zZ Q yyyy");
```

Comment:
This test was commented (for very long time, since 2012 or 2013): https://github.com/druid-io/druid/pull/4611/files/586d99cdea0e87f24a469ab681c167493d976a9a#diff-a7af8b2daf1212edc1fe51e80dc0b4dfL71 I uncommented it, but it was partially failing, and I removed the failing part. I think it was commented just because the test was failing. So this change at least doesn't reduce the test coverage.